src/WebJobs.Extensions.DurableTask/LinuxAppServiceFileLogger.cs

```diff
@@ -19,8 +19,9 @@ namespace Microsoft.Azure.WebJobs.Extensions.DurableTask
     /// We have modified their implementation to utilize syscall.rename instead of File.Move during file rolling.
     /// This change is necessary for older versions of fluent-bit, our logging infrastructure in linux dedicated, to properly deal with logfile archiving.
     /// </summary>
-    public class LinuxAppServiceFileLogger
+    internal class LinuxAppServiceFileLogger : IDisposable
```

Comment:
Remove this from public is a breaking API change - any concern? I'm not sure of the rules with this library.

Reply:
I believe this class was never meant for public usage, and in fact it's public-ness may be a problem for us. @cgillum: any thoughts?

Reply:
Agreed this should never have been public. I agree with changing it to be `internal`.